app/controllers/dashboard_controller.rb

```diff
@@ -140,8 +140,9 @@ def start_url
 
   def widget_chart_data
     widget = find_record_with_rbac(MiqWidget, params[:id])
-    datum = widget.contents_for_user(current_user).contents
-    content = nil
+    blank = widget.contents_for_user(current_user).blank?
```

Comment:
`widget.contents_for_user(current_user).` is querying the database (doing at least 3 selects),
so we should reuse the response instead of calling it twice in a row

Reply:
fixed